### ISSUE TITLE
InitError on julia 1.5

### ISSUE THREAD

**Issue by chengchingwen** [JuliaText/WordTokenizers.jl#55]
I get an `InitError` with WordTokenizers on Julia 1.5 when using WordTokenizers in a package. Even if the package is almost empty.

Here is an MWE
```julia
(@v1.5) pkg> generate TestToken

# in shell
$ cd TestToken/src
# add "using WordTokenizers" in TestToken.jl

$ julia --proj
(TestToken) pkg> add WordTokenizers

julia> using TestToken
[ Info: Precompiling TestToken [74092fa2-b961-4660-b22e-0e3f69c23db5]                                                     
ERROR: LoadError: InitError: Evaluation into the closed module `WordTokenizers` breaks incremental compilation because the
 side effects will not be permanent. This is likely due to some other module mutating `WordTokenizers` with `include` duri
ng precompilation - don't do this.                
Stacktrace:     
 [1] include(::Function, ::Module, ::String) at ./Base.jl:380      
 [2] include at ./Base.jl:368 [inlined]
 [3] include at /home/peter/.julia/packages/WordTokenizers/h9eVi/src/WordTokenizers.jl:2 [inlined]
 [4] __init__() at /home/peter/.julia/packages/WordTokenizers/h9eVi/src/WordTokenizers.jl:45
 [5] _include_from_serialized(::String, ::Array{Any,1}) at ./loading.jl:697
 [6] _require_search_from_serialized(::Base.PkgId, ::String) at ./loading.jl:782
 [7] _require(::Base.PkgId) at ./loading.jl:1007
 [8] require(::Base.PkgId) at ./loading.jl:928
 [9] require(::Module, ::Symbol) at ./loading.jl:923
 [10] include(::Function, ::Module, ::String) at ./Base.jl:380
 [11] include(::Module, ::String) at ./Base.jl:368
 [12] top-level scope at none:2
 [13] eval at ./boot.jl:331 [inlined]
 [14] eval(::Expr) at ./client.jl:467                    
 [15] top-level scope at ./none:3
during initialization of module WordTokenizers                     
in expression starting at /home/peter/peter/test/jlcode/TestToken/src/TestToken.jl:3
ERROR: Failed to precompile TestToken [74092fa2-b961-4660-b22e-0e3f69c23db5] to /home/peter/.julia/compiled/v1.5/TestToken
/VU1Wf_OdZ3m.ji.        
Stacktrace:                                         
 [1] error(::String) at ./error.jl:33
 [2] compilecache(::Base.PkgId, ::String) at ./loading.jl:1290
 [3] _require(::Base.PkgId) at ./loading.jl:1030
 [4] require(::Base.PkgId) at ./loading.jl:928
 [5] require(::Module, ::Symbol) at ./loading.jl:923


```

**Comment by oxinabox** [comment]
But not in Julia 1.0?
If so that is a julia bug and needs to be openned on JuliaLang/julia.

**Comment by chengchingwen** [comment]
Yes, I tried with Julia v1.0.5, v1.5.0, and also v1.5.1 . Only v1.0.5 seems to work fine.

**Comment by chengchingwen** [comment]
@oxinabox  This problem seems to be fixed with #56. I guess we might also need a ci server for testing julia v1.5 on this problem.
